src/Data/Swagger/Internal.hs

```diff
@@ -1050,6 +1050,10 @@ instance ToJSON (ParamSchema t) => ToJSON (SwaggerItems t) where
     [ "collectionFormat" .= fmt
     , "items"            .= schema ]
   toJSON (SwaggerItemsObject x) = object [ "items" .= x ]
+  toJSON (SwaggerItemsArray  []) = object -- Nullary schema, cf. <https://github.com/GetShopTV/swagger2/issues/167>.
```

Comment:
Can you make that comment into a haddock documentation for this instance and add a doctest?